services/openshift-service-impl/src/main/java/io/fabric8/launcher/service/openshift/impl/OpenShiftController.java

```diff
@@ -22,8 +22,7 @@
 
     @Override
     public boolean checkNamespace(String namespaceName) {
-        OpenShiftClient client = getOpenShiftClientOrNull();
-        try {
+        try (OpenShiftClient client = getOpenShiftClientOrNull()) {
```

Comment:
Closing the openshift client here is not a good idea, as it will be needed for other methods. This class will be removed soon, so don't bother changing it